uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/staff-leave-report.html

```diff
@@ -18,57 +17,59 @@
           </div>
         </div>
       </div>
-      <crm-loading show="!report.loading.content">
-        <div class="chr_leave-report__table table-responsive">
-          <table class="table table-condensed-mixed">
-            <thead>
-              <tr>
-                <th class="chr_leave-report__table__heading--label"></th>
-                <th class="chr_leave-report__table__heading--dates">Dates</th>
-                <th class="chr_leave-report__table__heading--status">Status</th>
-                <th
-                  class="chr_leave-report__table__heading--type"
-                  ng-repeat="absenceType in report.absenceTypesFiltered track by $index">
-                  {{absenceType.title}}
-                </th>
-                <th class="chr_leave-report__table__heading--action"></th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td
-                  class="chr_leave-report__label chr_leave-report__label--toggle"
-                  ng-click="report.toggleSection('entitlements')">
-                  <i
-                    class="fa"
-                    aria-hidden="true"
-                    ng-class="{ 'fa-chevron-down': report.sections.entitlements.open, ' fa-chevron-right': !report.sections.entitlements.open }"></i>
-                  Period Entitlement
-                </td>
-                <td></td>
-                <td></td>
-                <td ng-repeat="absenceType in report.absenceTypesFiltered track by $index">
-                  {{absenceType.entitlement}}
-                </td>
-                <td></td>
-              </tr>
-              <tr>
-                <td class="table-nested-cell" colspan="9">
-                  <crm-loading show="!report.sections.entitlements.loading">
-                    <div
-                      class="table-nested table-responsive"
-                      ng-show="report.sections.entitlements.open"
-                      ng-switch="!!report.sections.entitlements.data.length">
-                      <table class="table table-condensed" ng-switch-when="false">
-                        <tbody>
-                          <tr>
-                            <td align="center">No entitlement breakdown data available</td>
-                          </tr>
-                        </tbody>
-                      </table>
-                      <table class="table table-condensed" ng-switch-when="true">
-                        <thead>
-                          <tr>
+    <div class="chr_leave-report__table-container">
```

Comment:
why is the previous element padding bigger than this one if they are supposed to be at the same padding level?